networking_ccloud/ml2/mech_driver.py

```diff
@@ -51,12 +50,27 @@ def _plugin(self):
             self._plugin_property = directory.get_plugin()
         return self._plugin_property
 
-    def start_driver_rpc(self):
+    def start_rpc_listeners(self):
+        """Start the RPC listeners.
+
+        Most plugins start RPC listeners implicitly on initialization.  In
+        order to support multiple process RPC, the plugin needs to expose
```

Comment:
multiple process RPC? I like the wording in the commit-message more. This one, I don't understand.

Reply:
For the MechanismDriver I've been copying the original docstring into my file whenever I implement one of the methods, so this is taken from here: https://github.com/openstack/neutron-lib/blob/master/neutron_lib/plugins/ml2/api.py#L398-L404

You think a custom™ message would make more sense?

Reply:
hm ... then I don't understand the original and would have to take that confusion to upstream, I guess. No sure if a custom message is necessary, I just don't understand this one :D

Reply:
Maybe plugins can start multiple RPCWorker for different processes and that's all they're trying to tell you? But I won't haggle with upstream for a docstring myself, still trying to get the segment race condition fixed ;)